graphql_compiler/schema_generation/schema_graph.py

```diff
@@ -88,19 +89,15 @@ def get_superclasses_from_class_definition(class_definition):
     return []
 
 
+@six.add_metaclass(ABCMeta)
 class SchemaElement(object):
-    ELEMENT_KIND_VERTEX = 'vertex'
-    ELEMENT_KIND_EDGE = 'edge'
-    ELEMENT_KIND_NON_GRAPH = 'non-graph'
 
-    def __init__(self, class_name, kind, abstract, properties, class_fields):
+    @abstractmethod
```

Comment:
`__init__` is not abstract, it's implemented. This should get removed.

Reply:
https://stackoverflow.com/questions/44800659/python-abstract-class-init